README.md

```diff
@@ -22,14 +21,10 @@ A reasonable folder structure would be something like the following:
 yarn
 ```
 ## Setup contract node
-A specific version of the contract node is [required](https://stackoverflow.com/questions/69826769/how-can-i-query-contract-info-with-the-latest-polkadot-js-and-substrate-contract/69831057#69831057) to work with polkadotjs.
-
 ```bash
-git clone https://github.com/paritytech/substrate-contracts-node &&
 cd substrate &&
-git checkout 8d91b8e578065a7c06433cbd41ac059bf478a0bd &&
 cargo build && 
-./target/debug/substrate-contracts-node --dev --tmp --version
+./target/debug/substrate --dev --tmp --version
```

Comment:
`cargo run --release -- --dev --tmp --unsafe-ws-external --prometheus-external -lerror,runtime::contracts=debug`

Reply:
This is the command for the contracts node to run in `substrate`.